src/handlers.js

```diff
@@ -65,6 +65,27 @@ async function singlePackageListing(req, res, timecop) {
       og_image_height: 600,
     }});
   } catch(err) {
+    let status_to_display = false; // Since the status is ignored if not a number,
+    // we initialize as boolean to no-op in the case we don't find a proper status
+
+    const validStatus = (val, key) => {
```

Comment:
Very optional suggestion, but this function name was not obvious to me what it did, I dunno if this is any better, but **how about calling this function  `validStatusIs`?** or `statusIs`?

Since it is checking not just validity overall (based on first argument matching some "set" of okay values), but rather specifically matching to the exact supplied status (second argument).

So it is more "matching" to a supplied okay value, rather than "checking validity" only. (Very minor point, go with what you like.)